packages/grafana-ui/src/components/OptionsUI/fieldColor.tsx

```diff
@@ -36,7 +36,9 @@ export const FieldColorEditor: React.FC<FieldConfigEditorProps<FieldColor | unde
       description: mode.description,
       isContinuous: mode.isContinuous,
       isByValue: mode.isByValue,
-      component: () => <FieldColorModeViz mode={mode} theme={theme} />,
+      component: function render() {
+        return <FieldColorModeViz mode={mode} theme={theme} />;
+      },
```

Comment:
Similar comment to the previous one- how does this approach help the fixes?

Reply:
This is same like the previous one. I used regular function declaration to have the displayName.
But I just found this could be also written in the shorthand method style. I've changed this to make it consistent.  Thanks!